translation/models/translation_services.py

```diff
@@ -8,6 +8,24 @@
 from .translation import TranslatedStep, TranslatedLesson, TranslatedCourse, StepSource, TranslatedStepSource
 
 
+# :param: array of texts
+#  make array with escaped symbols (";", "&", "+")
+def escape_symbols(texts):
```

Comment:
@KirillMakarov actually we don't want to escape, we want ecode and `re.escape` act like this
`& -> \\ &` which is useless